### PR TITLE
[docs] Move density to accessibility page

### LINKED ISSUE #1303: [docs] Improve Accessibility page
I suggest we improve the [accessibility page](https://material-ui.com/components/data-grid/accessibility) in our documentation with the following changes: 

1. We move the density selector section and how it's great for some people
2. We cover all the keyboard shortcuts

### PR DESCRIPTION
**Pull request by dtassone**
fix #1303 

Preview: https://deploy-preview-1374--material-ui-x.netlify.app/components/data-grid/accessibility/#density

### CODE REVIEW

File: docs/src/pages/components/data-grid/accessibility/accessibility.md [hunk 1]
```diff
@@ -20,6 +20,26 @@ As meeting WCAG 2.0 level AA guidelines also meets the ADA and Section 508 stand
 
 The [WAI-ARIA authoring practices](https://www.w3.org/TR/wai-aria-practices/#grid) provides valuable insight on how to make the grid highly accessible.
 
+## Density
+
+You can change the density of the rows and the column header.
+
+### Density selector
+
+To enable the density selector you need to compose a toolbar containing the `GridDensitySelector` component, and apply it using the `Toolbar` key in the grid `components` prop.
```
Comment (oliviertassinari) [inline docs/src/pages/components/data-grid/accessibility/accessibility.md:29]: I wonder if this terminology is not more accurate, not sure.

```suggestion
To enable the density selector you need to compose a toolbar containing the `GridDensitySelector` component, and apply it using the `Toolbar` property in the grid `components` prop.
```

**Review by oliviertassinari — APPROVED** [review]

**Comment by oliviertassinari** [comment]
Could we wait about 24 hours so that Matt has the time to review the changes?

**Comment by dtassone** [comment]
> Could we wait about 24 hours so that Matt has the time to review the changes? I assume he will want to focus on how the sentences are turned.

I have not changed the content, I just moved the block around

**Comment by oliviertassinari** [comment]
True, it's was more an opportunity to polish it :)

**Comment by mbrookes** [comment]
I think we did previously, so all good.

**Pull request merged**